docs/howto/operator/gcp/_partials/prerequisite_tasks.rst

```diff
@@ -26,7 +26,7 @@ To use these operators, you must do a few things:
 
     .. code-block:: bash
 
-      pip install 'apache-airflow[gcp]'
+      pip install 'apache-airflow[google]'
```

Comment:
? This seems unrelated to this change?

Reply:
Ir's somewhat related. I could potentially separate it out, but I think having names of extras = provider package is the driver for it. Do you think it's worth separating? I can do it easily, but it requires changing setup.py and I would like to avoid changing it too often.

Reply:
Actually I realised there are more changes needed. I am extracting this change to a separate PR.